packages/docs/src/routes/docs/integrations/modular-forms/index.mdx

```diff
@@ -268,6 +268,35 @@ export default component$(() => {
 ```
 </CodeSandbox>
 
+## Dealing With Props
+
+Sometimes, you may need to initialize or update the state of your form from the parent component via props. In such cases, you can use the setValue function, as illustrated in the example below:
+
+```tsx
+import { setValue, useForm } from "@modular-forms/qwik";
+
+export interfaceFormProps {
```

Comment:
Hi @jermsam thanks for your help.
interfaceFormProps is unused, is this interface necessary for this snippet?

Reply:
thanks @gioboa - fixed and used